bundles/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/internal/handler/KeContactHandler.java

```diff
@@ -558,15 +557,17 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     if (command instanceof DecimalType) {
                         transceiver.send(
                                 "setenergy " + String.valueOf(
-                                        Math.min(Math.max(0, ((DecimalType) command).intValue()*10), 999999999)), this);
+                                        Math.min(Math.max(0, ((DecimalType) command).intValue() * 10), 999999999)),
+                                this);
```

Comment:
what's with this new indentation?

Reply:
The Eclipse auto save formatting placed it on a new line. Does it also happen for you?

Reply:
I don't mind that it placed it on a new line, I just disagree on the indentation of that new line, it should be at least as indented as the previous one.

Reply:
I stopped fighting auto formatters a long time ago. :wink:

According to my settings the limit is set to 120:

![settings](https://user-images.githubusercontent.com/12213581/78469159-ca00dd80-771e-11ea-8547-897d8730a15f.png)

It's a limit for a reason, sometimes it will result in less optimal formatting like in this case. It's expected to happen a lot more when all code is reformatted using spotless.

Reply:
The auto formatter indents it like this because it is the second parameter. I think it does make all this excessively indented code more readable:

![indentation](https://user-images.githubusercontent.com/12213581/78470340-f7eb1f80-7728-11ea-8166-08515a982858.png)